forem/forem
# Confirmations that users GDPR data was deleted

## What type of PR is this? (check all applicable)

## Description
- list of internal notifications for admins to destroy users gdpr data that should replace slack notifications
- admins are able to confirm that the data was deleted, then the notification is destroyed

Removing slack notifications is the next step.

## Related Tickets & Documents
https://github.com/forem/forem/projects/31#card-45606967

## QA Instructions, Screenshots, Recordings
List of the users whose gdpr data should be deleted:
![изображение](https://user-images.githubusercontent.com/30115/97019961-a5a10d00-1559-11eb-9438-3f57c0a7ffd8.png)
A new record in this list should appear after a user profile is destroyed by themselves or by an admin.

Notice after user delete (by admin):
![изображение](https://user-images.githubusercontent.com/30115/97022835-4a711980-155d-11eb-9af6-95fbd8bda096.png)

## Added tests?
